### ISSUE TITLE
SRTM endpoint return 404

### ISSUE THREAD

**Issue by jpenninkhof** [tkrajina/srtm.py#51]
It seems that the SRTM endpoint have been removed. All of them return code 404 at the moment.

**Comment by tkrajina** [comment]
Yes, they changed the URLs and the way the data is downloaded. It now requires a basic html authentication.

Once I fix https://github.com/tkrajina/go-elevations/issues/3 I'll move to srtm.py to make it work again. Unfortunately my free time for this is limited, so this (probably) won't be ready in a couple of days :( But I'll try my best to do it next week.

**Comment by farawayhillsaregreener** [comment]
Seconding @jpenninkhof on this issue, and issuing a thanks in advance!

**Comment by jgliss** [comment]
It would be very helpful if this could be fixed. Many thanks in advance @tkrajina !

**Comment by tkrajina** [comment]
@jgliss @farawayhillsaregreener @jpenninkhof I'm still working on the earthdata branch, but as a temporary solution I changed master to use a SRTM mirror provided by https://srtm.kurviger.de

So, `master` works now. I also released a new version on pypi. 

I'm leaving this issue open until I also fix and merge the earthdata branch.

**Comment by jgliss** [comment]
@tkrajina great thanks!
